_posts/2021-12-14-strimzi-and-log4shell.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: post
+title:  "Strimzi and Log4Shell (Log4j2 CVE-2021-44228)"
+date: 2021-12-14
+author: jakub_scholz
+---
+
+The [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228) vulnerability in the [Log4j2 logging library](https://logging.apache.org/log4j/2.x/) - also known as Log4Shell - affects many software projects written in Java.
+Several Strimzi components and dependencies use Log4j2 as well.
+A lot has been written about how the vulnerability works and how it can be used by attackers to gain control over your system.
+We will not get into the details in this blog post.
+But in short, if an attacker can get your application to log some arbitrary log message (or part of it), it can be used to execute arbitrary code, loaded from an attacker-controller remote server, inside your application.
+In this blog post, we will have a look at which parts of your Strimzi deployment might be affected and how the vulnerability can be mitigated.
+
+<!--more-->
+
+## Is Strimzi affected?
+
+When using Strimzi, you first deploy the Cluster Operator as the central component.
+And then you use it to deploy the operands.
```

Comment:
What about dropping `And` and following on the line below with something like `finally each operand ...` ?